packages/account/src/Sections/Security/Passkeys/components/passkey-removing-verification.tsx

```diff
@@ -0,0 +1,22 @@
+import { Localize } from '@deriv-com/translations';
+// import { mobileOSDetect } from '@deriv/shared';
+import { DerivLightIcVerifyPasskeyIcon } from '@deriv/quill-icons';
+import { PasskeysStatusLayout, TPasskeysButtonOnClicks } from './passkeys-status-layout';
+
+// TODO: edit the component and add tests when e-mail flow is implemented
+export const PasskeyRemovingVerification = ({ onPrimaryButtonClick }: TPasskeysButtonOnClicks) => {
+    // TODO: edit the description considering OS
+    return (
+        <div className='passkeys'>
+            <PasskeysStatusLayout
+                description={
+                    <Localize i18n_default_text="We'll send you a secure link to verify your request. Tap on it to confirm you want to remove the passkey. This protects your account from unauthorised requests." />
+                }
+                icon={<DerivLightIcVerifyPasskeyIcon height='96px' width='96px' />}
+                title={<Localize i18n_default_text='Verify your request' />}
+                onPrimaryButtonClick={onPrimaryButtonClick}
+                primary_button_text={<Localize i18n_default_text='Send email (back to list)' />}
```

Comment:
what does "back to list" mean?

Reply:
this is just for testing, this component is no used in current flow. should be refactored once email verification is implemented on BE side